testsuite/src/test/java/org/jboss/intersmash/tests/wildfly/web/cache/offload/infinispan/WildflyOffloadingSessionsToInfinispanApplication.java

```diff
@@ -63,6 +63,8 @@ public WildflyOffloadingSessionsToInfinispanApplication() {
 				.withValue(Infinispan2ReplicasService.INFINISPAN_CUSTOM_CREDENTIALS_USERNAME).build());
 		environmentVariables.add(new EnvVarBuilder().withName("CACHE_PASSWORD")
 				.withValue(Infinispan2ReplicasService.INFINISPAN_CUSTOM_CREDENTIALS_PASSWORD).build());
+		environmentVariables.add(new EnvVarBuilder().withName("TRUST_STORE_PATH")
```

Comment:
I don't think this is needed in WildflyOffloadingSessionsToInfinispanApplication, since that class configures an application that uses [the Kubernetes auto-monted service account truststore](https://github.com/Intersmash/intersmash-applications/blob/main/wildfly/web-cache-offload-infinispan/scripts/script.cli#L25)

Reply:
Here we are using a self-signed certificate and WF needs to know it has to trust it